src/api/public/apidocs-new/oauth2-redirect.html

```diff
@@ -67,9 +67,13 @@
         window.close();
     }
 
-    window.addEventListener('DOMContentLoaded', function () {
-      run();
-    });
+    if (document.readyState !== 'loading') {
```

Comment:
What is the reason for this change?

Reply:
You may want to have look at the [upstream project](https://github.com/swagger-api/swagger-ui/commit/3eb05287efec9300ae612092e44f5378113179a3).

I documented how we update Swagger UI in [our wiki](https://github.com/openSUSE/open-build-service/wiki/New-Swagger-API-doc#how-to-update). Please have a look.

Reply:
Oh okay, this is from upstream :+1: